components/ide/jetbrains/backend-plugin/gradle-stable.properties

```diff
@@ -1,9 +1,9 @@
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=222.4459
+pluginSinceBuild=222.4345
```

Comment:
This update is incorrect, and we already have a PR to fix the auto-updater:
- https://github.com/gitpod-io/gitpod/pull/14738